Career Endorsement Tracker/Career Endorsement Tracker/ViewControllers/Home/HomeDetailTableViewCell.swift

```diff
@@ -20,23 +20,29 @@ class HomeDetailTableViewCell: UITableViewCell {
             updateViews()
         }
     }
+    let completedImage = UIImage(named: "checkBox")
+    let incompleteImage = UIImage(named: "uncheckedBox")
     
     private func updateViews() {
         guard let step = step else {
             print("No step")
             return
         }
         stepTextView.text = step.steps_description
+        stepTextView.sizeToFit()
         if step.is_complete {
             submitCompletionButton.backgroundColor = .green
-            submitCompletionButton.setTitle("Completed", for: .normal)
+            submitCompletionButton.setImage(completedImage, for: .normal)
+//            submitCompletionButton.setImage("checkedBox, for: .normal)
+//            submitCompletionButton.setTitle("Completed", for: .normal)
         } else {
             submitCompletionButton.backgroundColor = .red
-            submitCompletionButton.setTitle("Not Completed", for: .normal)
```

Comment:
The un-clicked button works properly.  Is there something different between the two states that would prevent the click from working?